docs/source/app-dev/app-arch.rst

```diff
@@ -1,295 +1,121 @@
 .. Copyright (c) 2020 The DAML Authors. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Application architecture guide
-#########################################
+Application architecture
+########################
 
-This document is a guide to building applications that interact with a DA ledger deployment (the 'ledger'). It:
+This section describes our recommended design of a full-stack DAML application.
 
-- describes the characteristics of the ledger API, how this affects the way an application is built (the 'application architecture'), and why it is important to understand this when building applications
-- describes the resources in the SDK to help with this task
-- gives some guidelines to help you build correct, performant, and maintainable applications using all of the supported languages
+.. image:: ./recommended_architecture.svg
 
-Categories of application
-*************************
+The above image shows the recommended architecture. If you are working with a different tech stack DAML SDK also supports :ref:`Java <java-bindings>` and :ref:`Scala
```

Comment:
I think it’s worth mentioning here that the JSON API can be used from any language and given that basically every dev is familiar with making HTTP requests this is a much smaller step than trying to use the gRPC API.

Reply:
agreed! I rearranged and added a sentence.